src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -433,7 +436,12 @@ export class UBSOrderDetailsComponent extends FormBaseComponent implements OnIni
     }, 0);
   }
 
+  canAddEcoShopOrderNumber(): boolean {
+    return this.ecoShopOrdersNumbersCounter < this.limitOfEcoShopOrdersNumbers ? true : false;
```

Comment:
Do you really need these true and false? Is it possible to return just this.ecoShopOrdersNumbersCounter < this.limitOfEcoShopOrdersNumbers?

Reply:
Really, l don't think about it, thank you for advice, l'll fix it and push changes!